osu.Game.Rulesets.Catch/UI/DrawableCatchRuleset.cs

```diff
@@ -52,5 +54,7 @@ private void load()
         protected override PassThroughInputManager CreateInputManager() => new CatchInputManager(Ruleset.RulesetInfo);
 
         public override DrawableHitObject<CatchHitObject>? CreateDrawableRepresentation(CatchHitObject h) => null;
+
+        protected override ResumeOverlay CreateResumeOverlay() => new DelayedResumeOverlay { Scale = new Vector2(0.65f) };
```

Comment:
Was `0.65f` derived from something or is that a throwaway figure?

Reply:
I forgot to comment on this, but it's an eyeballed value that brings the size of it in line with other rulesets. Looks like the playfield adjustment container is scaling things up here.

I'll either comment on it, or remove it if my above proposal is accepted (because it doesn't look too bad there).